### DISCUSSION TITLE
Has this been abandoned? No updates for some time now

### DISCUSSION THREAD

**Discussion by NewbieCPlusPlus** [bromite/bromite#2573] [General]
Hello, I was just wondering has this project been abandoned because I havent seen an update for more than two and a half months now. Latest stable Chromium released on 27th of January. I was wondering is it safe to use Bromite now with no latest security fixes?

**Comment by ameyparab3** [comment]
I don't care much about security since I only browser surface websites. But I can't even download the browser on f-droid giving error cannot verify the file.

**Comment by kkkrist** [comment]
If it was my project and I'd see some of the poop dumped into the issue tracker, I'd definitely abandon it. Luckily @csagan5 does not seem to get worked up as easily as me, so their reasons for taking time out are probably external to the project. Maybe we should just respect that.

**Reply by Savasdotexe** [reply]
What "poop" dumped into the issue tracker are you talking about? I've not looked in detail, but this seems pretty snobby.

**Reply by eclairevoyant** [reply]
@Savasdotexe Maybe look next time before commenting?
Here's a "great" example: https://github.com/bromite/bromite/issues/2583

Frankly I would ban such users but the maintainers here are far more patient.

**Reply by fohfuu** [reply]
> If it was my project and I'd see some of the poop dumped into the issue tracker, I'd definitely abandon it. Luckily @csagan5 does not seem to get worked up as easily as me, so their reasons for taking time out are probably external to the project. Maybe we should just respect that.

This is a neutral question, asking about abandonment due to a valid security issue that's been open for months. "It's the responsibility of the user to make sure it's safe to use", as you said below. What did they say that implied disrespect?

Entitlement shown by some in the community is unacceptable, I agree. However, misaimed criticism can discourage a user from doing their due diligence for fear of inadvertently causing offence. We can show respect to each other by assuming reasonable questions are being asked in good faith.

**Reply by salad612** [reply]
> > If it was my project and I'd see some of the poop dumped into the issue tracker, I'd definitely abandon it. Luckily @csagan5 does not seem to get worked up as easily as me, so their reasons for taking time out are probably external to the project. Maybe we should just respect that.
> 
> This is a neutral question, asking about abandonment due to a valid security issue that's been open for months. "It's the responsibility of the user to make sure it's safe to use", as you said below. What did they say that implied disrespect?
> 
> Entitlement shown by some in the community is unacceptable, I agree. However, misaimed criticism can discourage a user from doing their due diligence for fear of inadvertently causing offence. We can show respect to each other by assuming reasonable questions are being asked in good faith.

I don't think they were talking about this discussion specifically. More like #2583

**Reply by fohfuu** [reply]
> > > If it was my project and I'd see some of the poop dumped into the issue tracker, I'd definitely abandon it. Luckily @csagan5 does not seem to get worked up as easily as me, so their reasons for taking time out are probably external to the project. Maybe we should just respect that.
> > 
> > 
> > This is a neutral question, asking about abandonment due to a valid security issue that's been open for months. "It's the responsibility of the user to make sure it's safe to use", as you said below. What did they say that implied disrespect?
> > Entitlement shown by some in the community is unacceptable, I agree. However, misaimed criticism can discourage a user from doing their due diligence for fear of inadvertently causing offence. We can show respect to each other by assuming reasonable questions are being asked in good faith.
> 
> I don't think they were talking about this discussion specifically. More like #2583

That's kind of my point. Asking that "maybe we should just respect" that csagan5 may have external reasons for not updating implies there's a level of _dis_respect in asking about the status of the project.

**Reply by salad612** [reply]
there isn't. there however is disrespect in using explicatives like how they were used in the mentioned issue. especially when there were already issues made beforehand.

**Reply by Savasdotexe** [reply]
> @Savasdotexe Maybe look next time before commenting? Here's a "great" example: #2583
> 
> Frankly I would ban such users but the maintainers here are far more patient.

How could I possibly see that post when it happened after I posted my comment? Ironic that you're telling me to look before commenting.

I would ban them too, maybe the lack of moderators is a factor more than being patient.

Anyway, every project gets bad apples, and the logic here is that he's possibly taking time out because of the "poop" being posted, so I wanted to know what he was referring to, not some abusive person due that posted last week because of the gap in updates.

**Reply by fohfuu** [reply]
> there isn't. there however is disrespect in using explicatives like how they were used in the mentioned issue. especially when there were already issues made beforehand.

Those issues are bad. However, "is this project active?" is not an issue. It is good practice to ask broad questions in the discussion forum if it is not answered elsewhere.

The original post is not a rude question and it is not disrespectful to the maintainer to ask about the lack of updates. Therefore, it is inappropriate to mention a rude issue and say we should be more respectful in reply to this post.

I'm not going to repeat this over and over, but it does bare stressing, because discouraging responsible users from politely asking security questions is dangerous. I'll leave it at that.

**Reply by salad612** [reply]
> > there isn't. there however is disrespect in using explicatives like how they were used in the mentioned issue. especially when there were already issues made beforehand.
> 
> Those issues are bad. However, "is this project active?" is not an issue. It is good practice to ask broad questions in the discussion forum if it is not answered elsewhere.
> 
> The original post is not a rude question and it is not disrespectful to the maintainer to ask about the lack of updates. Therefore, it is inappropriate to mention a rude issue and say we should be more respectful in reply to this post.
> 
> I'm not going to repeat this over and over, but it does bare stressing, because discouraging responsible users from politely asking security questions is dangerous. I'll leave it at that.

you are Right

**Comment by mpaunovic** [comment]
https://en.wikipedia.org/wiki/Bus_factor

The Bus Factor is a dead giveaway of a bad project management system.

Hypothetical situation: Let's say something happens to csagan5 (God forbid!). Now what? How will/can the project survive?

**Comment by kkkrist** [comment]
Let's not forget that open source maintainers working in their free time don't owe their users anything. There is no contract stipulating any kind of support or ongoing development, it's not a commercial product. Also it's the responsibility of the user to make sure it's safe to use.

**Reply by Mr-Bajs** [reply]
True. And part of the user responsibility lies by asking about the status of the project. There is no demands in asking.

**Comment by mpaunovic** [comment]
Sorry, but it looks like you're missing the point here. It's not about open source maintainers owing their users anything, it's about their users owing them something. For starters, constructive criticism. And, yes, people can be jerks (surprise, surprise!) and there are some difficult discussions around holding back the community rather than allowing it to flourish, but that doesn't change the fact that the vast majority of Bromite users are simply trying to help the project.

**Comment by GameOverFlowChart** [comment]
"Now what? How will/can the project survive?"
The project is opensource, someone else can fork it and continue the work. This happens often with opensource projects. 

Also I came here to say that my F-droid client can't sync with the repo anymore.

**Reply by nad0vs** [reply]
> "Now what? How will/can the project survive?" The project is opensource, someone else can fork it and continue the work. This happens often with opensource projects.
> 
> Also I came here to say that my F-droid client can't sync with the repo anymore.

Turns out the fdroid repo issue is known and devs are working on it https://github.com/bromite/bromite/issues/2575

**Comment by salad612** [comment]
They're Back!
❤️

**Reply by camm9909** [reply]
Since when lol. Project has been abandoned for weeks. Any alternative browsers?

**Reply by salad612** [reply]
@ckbaudio 
csagan has been active recently so hopefully there will be updates soon.https://github.com/bromite/bromite/issues/2575#issuecomment-1435941868

 Brave is a good browser when you disable the bloat. Or use Mull with uBlock Origin

**Comment by AMoll1** [comment]
https://github.com/uazo/bromite-buildtools/releases

**Comment by Harrier-Du-Bois** [comment]
Seeing how the last official build is quite outdated by now, I find it somewhat irresponsible to not mention this fact one way or another on the official website, bromite.org

Personally I like to see the site is still up as it keeps a shred of hope alight that this project will see a comeback, but if it was my responsibility I'd put a disclaimer somewhere on the top of the page.

No idea if @csagan5 might see this, so maybe @uazo could forward the suggestion, provided he is in contact.

No hard feelings, wishing everyone all the best and crossing my fingers for a return of Bromite.

**Reply by salad612** [reply]
There is a prompt in the app that links to this page https://www.bromite.org/vulnerable-version
but maybe it could be on the front page as well

**Reply by Harrier-Du-Bois** [reply]
Yeah, that's true.
I was more thinking in terms of new users who might happen to find out about Bromite just now.
Even this git directs them to that main page and there's no real mention of the halt in dev, so some people might get a false impression...

![IMG_20230506_132311](https://user-images.githubusercontent.com/89971442/236621255-d4423e86-d5d1-4cb4-bd46-1c7f5a970142.jpg)

Like I said, it's just a friendly suggestion and I never assumed anybody tried to hide the state of things out of spite or malice. But as we here are all rather security-minded people, I stand by finding it a bit irritating to leave that page the way it is, knowing the potential risks.

**Comment by Clouud9** [comment]
@csagan5 Please don't abandon this if you can! Bromite is one of the best browsers I have ever used, and I'd hate to see it be abandoned!

**Comment by elecvey** [comment]
Really, it's the only adequate browser among others. Which has such extensive features that even extensions are unnecessary, but is not overflowing with bloatware. I use it for years.  
I really appreciate your work, @csagan5, @uazo
Please, don't make your project lost in the past!

**Comment by GameOverFlowChart** [comment]
Maintaining such an open source product isn't easy for a single dev, if he doesn't even have the time to answer (or doesn't feel like answering) we should let him his rest. The good thing is it's open source so others can fork it and as far as I know someone already did. I'm using Mulch and Fulguris as alternatives for now but they a don't fulfill all my requirements.

**Reply by ghost** [reply]
I'm using vanadium...works just as good

**Reply by hank-tu-biz** [reply]
Is there a way to get Vanadium is you just have regular android? I can't seem to find it anywhere.

**Comment by TurtlesAI** [comment]
Mull and Mulch are nice, but Bromite was better.

**Reply by hank-tu-biz** [reply]
Agree. I'm using Iceraven since Bromite is gone.

**Reply by ghost** [reply]
No, you have to use Graphene OS to get vanadium. TBH, caring about privacy and using stock android is an oxymoron. Each and every single thing you do on stock android gets routed through google play services and ends up at google headquarters, including your keystrokes, even when you use bromite. See link below: 

https://digitalcontentnext.org/blog/2018/08/21/google-data-collection-research/

It's disgusting what they collect.

**Reply by fohfuu** [reply]
The report you link here doesn't support your point.

No love for Google here, but this paper studies a "LG X Power device with Android 6.0 version installed". A phone model from 2016, using a version of Android that had support dropped 3 years before the experiment, and 8 years ago from present. There is no way it could possibly reflect the modern data collection practices of Google, which have likely changed significantly in purpose, transparency, and volume. Probably not for the better! That's not to mention the various other glaring procedural problems and the blatant bias in the writing. It wasn't a useful case study at the time, and it has no relevance today.

Google is hardly so unsuspicious that you need to rely on aesthetically science-themed PDFs from 2018 to prove they harvest data. Try searching "google play services" in Elsevier, Springer, or (gasp!) Google Scholar and filter by date, then access the relevant results (using sci-hub if necessary) with a critical eye. Don't trust everything that agrees with you.

This is solely a debunking of the bad information provided. I am not continuing this conversation because it's veering wildly off-topic.

**Comment by GameOverFlowChart** [comment]
https://github.com/uazo/cromite
